auth/credentials/detect_test.go

```diff
@@ -805,26 +804,29 @@ func TestDefaultCredentials_UniverseDomain(t *testing.T) {
 		{
 			name: "service account json with file universe domain",
 			opts: &DetectOptions{
-				CredentialsFile:  "../internal/testdata/sa_universe_domain.json",
-				UseSelfSignedJWT: true,
+				CredentialsFile: "../internal/testdata/sa_universe_domain.json",
+				// default scopes are set in resolveDetectOptions before calling DetectDefault.
+				Scopes: []string{"https://www.googleapis.com/auth/cloud-platform"},
```

Comment:
It seems like there are no test cases that manually set UseSelfSignedJWT= true now. Can you add one for the GDU case.

Reply:
This test (`TestDefaultCredentials_UniverseDomain`) is just for universe domain, but `TestDefaultCredentials_ServiceAccountKeySelfSigned` does set UseSelfSignedJWT= true for GDU.